### PR TITLE
Placedog (Independent Publisher)

### PR DESCRIPTION
**Pull request by troystaylor**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

brave_whrUf1CGGN
brave_u3Pm3Ooxhx

### CODE REVIEW

File: independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,7047 @@
+{
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json:1]: @troystaylor Please remove artifacts related to other connectors from this PR.

File: independent-publisher-connectors/Placedog/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Placedog
+Request dogs from placedog.net. Hand-picked, stunning photographs of our favorite pets: dogs! Free to use with your websites/projects! Featured in the #ThePawryParty Chrome Extension!
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Placedog/readme.md:2]: @natalie-pienkowska Can you please let me know if I can go ahead with this submission ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json (hunk 1 above)
Comment (troystaylor) [reply independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json]: Done

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@troystaylor We are reviewing your connector again with management to move forward.

File: independent-publisher-connectors/Placedog/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{width}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Placedog/apiDefinition.swagger.json:26]: @troystaylor Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: independent-publisher-connectors/Placedog/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{width}": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Placedog/apiDefinition.swagger.json:24]: @troystaylor Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger.

File: independent-publisher-connectors/Placedog/apiDefinition.swagger.json (hunk 3 above)
Comment (sriyen-msft) [reply independent-publisher-connectors/Placedog/apiDefinition.swagger.json:26]: @troystaylor Applies the same comment for empty schemas in the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@troystaylor Apologies for the delay in response.
Update - Your connector is now in production deployment and reached [Test](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#region-deployment) region, and it would be completed and available in all the production regions within 3 weeks. Please let us know if you have any questions, Thanks.

**Comment by sriyen-msft** [comment]
@troystaylor Congratulations! The deployment of your connector was already completed on 02/07/2022,  Apologies for the delay in response. We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**